pkg/users/local.js

```diff
@@ -271,6 +272,30 @@ function is_user_in_group(user, group) {
     return false;
 }
 
+var AccountItem = React.createClass({
+    displayName: 'AccountItem',
+    click: function() {
+        cockpit.location.go([this.props.name]);
+    },
+    render: function() {
+        return React.createElement('div', {className: "cockpit-account", onClick: this.click },
+            React.createElement('div', {className: "cockpit-account-pic pficon pficon-user"}),
+            React.createElement('div', {className: "cockpit-account-real-name"}, this.props.gecos),
+            React.createElement('div', {className: "cockpit-account-user-name"}, this.props.name)
+        );
+    }
+});
+
+var AccountList = React.createClass({
+    displayName: 'AccountList',
+    render: function() {
+        var i, items = [];
+        for (i in this.props)
```

Comment:
Wouldn't it be more react-like to have a named property here, e.g. `this.props.accounts`? This feels like a hack.